lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -608,7 +608,7 @@
 '''
 
 RETURN = '''
-ansible_docker_container:
+docker_container:
```

Comment:
Hmm, like this : https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/docker/docker_container.py#L630 ? 

apparently suggested change has already been commited : 
https://github.com/ansible/ansible/commit/20bf02f6b96356ab5fe68578a3af9462b4ca42a5#diff-ff29a0f2b632e64441519bf4d9c578d3